demos/browser/app/meeting/meeting.ts

```diff
@@ -180,7 +180,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
           }
           (document.getElementById(
             'meeting-id'
-          ) as HTMLSpanElement).innerHTML = `${this.meeting} (${this.region})`;
+          ) as HTMLSpanElement).innerHTML += ` ${this.meeting} (${this.region})`;
```

Comment:
Why not just assign the whole string once here?

Reply:
The [joinInfo scope is local to the authenticate()](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L1054) which is being passed as a parameter to the [initializeMeetingSession()](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L556)
If i have to use the meetingid [here](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L183) then i need to create an instance variable. Thus changing the scope of the joinInfo.
However i can remove [this line from here](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L183) and use it in the [initializeMeetingSession()](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L556) but that would change the [initEventListeners()](https://github.com/aws/amazon-chime-sdk-js/blob/96e922e175b6e654486d14d7bb79f6fc4ae9c4d8/demos/browser/app/meeting/meeting.ts#L157)

Reply:
You can pass the meeting ID back as the result of authenticate().